julia.html.markdown

```diff
@@ -669,14 +669,14 @@ Lion    <: Cat  # => true
 Panther <: Cat  # => true
 
 # Defining a function that takes Cats
-function pet_cat(cat::Cat)
+function petCat(cat::Cat)
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194